app/controllers/reports_controller.rb

```diff
@@ -92,22 +98,19 @@ def find_report_type
     @report_type = reports_params[:report_type]
   end
 
-  def all_child_organizations(org_ids)
+  def all_child_organizations_with_self(org_ids)
     org_ids = org_ids.compact
-    if org_ids.empty?
-      []
-    else
-      orgs = Organization.where(parent_id: org_ids)
-      orgs | all_child_organizations(orgs.pluck(:id))
+    result = []
+    unless org_ids.empty?
+      result << org_ids
+      
+      orgs = Organization.find(org_ids)
+      orgs.each do |org|
+        result << org.all_child_organizations.pluck(:id)
```

Comment:
How can you remove the `all_child_organizations` method and still call it here?

Reply:
> How can you remove the `all_child_organizations` method and still call it here?

On line 79 of the Organization model, a method called `all_child_organizations` exists.  What Shirley did was create controller methods with the same name and try and call those.  I partly refactored that code in order to remove that ambiguity.  The code words as intended otherwise.